inputformat/src/main/java/org/zuinnote/hadoop/bitcoin/format/mapred/AbstractBitcoinFileInputFormat.java

```diff
@@ -59,7 +58,7 @@ public void configure(JobConf conf) {
 */
 @Override
   protected boolean isSplitable(FileSystem fs, Path file) {
-    if (!(this.isSplitable)) {
```

Comment:
this might be seen as a typo, but the function isSplitable is called like this in Hadoop. Hence, I propose we call the variable the same....

Reply:
Acknowledged. Makes sense!